config/locales/en.yml

```diff
@@ -22,3 +22,6 @@
 en:
   hello: "Hello world"
   sensitive_message: "Culturally sensitive content: This is an image of a person or persons now deceased. In some Aboriginal Communities, hearing names or seeing images of deceased persons may cause sadness or distress, particularly to the relatives of these people.  Would you like to view its content?"
+  date_labels: {
```

Comment:
This looks great. However, can we simply add all 5 type/label combinations in here? Then we have a single place to point our metadata staff to, should they want changes made.